app/src/main/java/com/quran/labs/androidquran/util/VoiceCommandsUtil.java

```diff
@@ -0,0 +1,96 @@
+package com.quran.labs.androidquran.util;
+
+import android.content.Context;
+import android.content.Intent;
+import android.widget.Toast;
+import com.quran.labs.androidquran.AboutUsActivity;
+import com.quran.labs.androidquran.HelpActivity;
+import com.quran.labs.androidquran.QuranPreferenceActivity;
+import com.quran.labs.androidquran.R;
+import com.quran.labs.androidquran.data.QuranInfo;
+import com.quran.labs.androidquran.ui.PagerActivity;
+import com.quran.labs.androidquran.ui.QuranActivity;
+
+public  class VoiceCommandsUtil {
+
+  private final Context myContext;
+
+  public VoiceCommandsUtil(Context cont) {
```

Comment:
@WailB parameter name should be written as full name (`context`) not abbreviation one (`cont`)